packages/utils/src/set-pixel-alt.ts

```diff
@@ -0,0 +1,20 @@
+import { BuilderContent, BuilderElement } from '@builder.io/sdk';
+import traverse from 'traverse';
+
+const isBuilderPixel = (item: unknown): item is BuilderElement => {
+  return (item as any).id?.startsWith('builder-pixel');
+};
+
+export function setPixelAlt(content: BuilderContent, alt: string): BuilderContent {
```

Comment:
Can we make this more generic (or add another function) and accept an object with properties to set? For example, another value people want to set on pixels is `'aria-hidden': 'true',`